src/omotes_orchestrator/sdk_interface.py

```diff
@@ -130,6 +131,8 @@ def connect_to_job_submissions(
             self.broker_if.add_queue_subscription(
                 OmotesQueueNames.job_submission_queue_name(workflow_type),
```

Comment:
Nitpicking: maybe also with keyword `queue_name=` when passing this argument to keep the consistency here and also in other `self.broker_if.add_queue_subscription()` below.

Reply:
Fixed: https://github.com/Project-OMOTES/orchestrator/pull/56/commits/d7501ccfa594ba4d19cbe5f3e03182e4c2b151d5